src/libs/actions/Session/index.js

```diff
@@ -187,6 +187,8 @@ function beginSignIn(login) {
             value: {
                 ...CONST.DEFAULT_ACCOUNT_DATA,
                 isLoading: true,
+                errors: null,
```

Comment:
Hmm, `errors` is already null in the default data, may be we just need  `message` field